porcupine/plugins/filemanager.py

```diff
@@ -217,10 +283,17 @@ def is_NOT_project_root(path: Path) -> bool:
     return path not in map(get_path, get_directory_tree().get_children())
 
 
+def can_paste(path: Path) -> bool:
+    return is_NOT_project_root(path) and copy_path is not None and copy_path.is_file()
+
+
 commands = [
     # Doing something to an entire project is more difficult than you would think.
     # For example, if the project is renamed, venv locations don't update.
     # TODO: update venv locations when the venv is renamed
+    Command("Copy", "<<Copy>>", (lambda p: not p.is_dir()), copy),
+    Command("Paste", "<<Paste>>", can_paste, paste),
+    Command("Paste here", None, (lambda p: p.is_dir() and copy_path is not None and copy_path.is_file()), paste_here),
```

Comment:
The condition is a bit long. It might be nicer to put it into a function, similarly to `can_paste()`.

Reply:
Sure